README.md

```diff
@@ -1,11 +1,11 @@
 # WSO2 Enterprise Integrator Kubernetes Artifacts 
 *Kubernetes Artifacts for container-based deployments of WSO2 Enterprise Integrator*
 
-
-This initial version contains the deployment of a single integrator container instance 
-running with one local [h2] registry database and one external [MySQL] user management database.
+This initial version contains the deployment of a single, but "scalable" WSO2 EI integrator container instance, 
```

Comment:
I think it's better not to say a single instance here as it is scalable to n number of instances. Thanks!